fmt/colors.ts

```diff
@@ -134,6 +134,11 @@ export function bold(str: string): string {
 /**
  * The text emits only a small amount of light.
  * @param str text to dim
+ *
+ * ## Warning
+ *
+ * Not all terminal emulators support `dim`.
```

Comment:
```suggestion
 * Warning: Not all terminal emulators support `dim`.
```
I think this should be fine.